packages/orchestration/src/examples/unbondExample.contract.js

```diff
@@ -48,10 +48,11 @@ export const start = async (zcf, privateArgs) => {
       // keeping it focused on long-running operations.
       const celestia = await orch.getChain('celestia');
       const celestiaAccount = await celestia.makeAccount();
+      const celestiaStaking = celestiaAccount.getStakingFacet();
```

Comment:
I think we want the staking operations on the account object.  It's certainly as legit as `liquidStake` as a custom operation on the stride account